documentation/modules/oauth/con-oauth-reauthentication.adoc

```diff
@@ -1,11 +1,12 @@
 // Module included in the following assemblies:
 //
-// assembly-oauth.adoc
+// assembly-oauth-authentication.adoc
 
 [id='con-oauth-reauthentication-{context}']
 = Session re-authentication for Kafka brokers
 
-The Kafka _SASL OAUTHBEARER_ mechanism, which is used for OAuth 2.0 authentication in Strimzi, supports a Kafka feature called the _re-authentication_ mechanism.
+The _SASL OAUTHBEARER_ authentication mechanism supports a Kafka feature called the _session re-authentication_ mechanism. 
+Session re-authentication is not supported if _SASL_PLAIN_ authentication is used.
```

Comment:
There is some nuance to this. And maybe we could better explain it. Re-authentication happens in both cases. `maxSecondsWithoutReauthentication` setting sets the maximum session timeout regardless of the mechanism.
The difference between OAUTHBEARER behavior and PLAIN behavior is that for OAUTHBEARER the token expires after some time, and that time takes precedence if shorter than the configured `maxSecondsWithoutReauthentication`, whereas for PLAIN the re-authentication is forced at the configured time. The same `maxSecondsWithoutReauthentication` config is used for both OAUTHBEARER and PLAIN on the same listener. If `maxSecondsWithoutReauthentication` is not set then PLAIN clients can stay connected indefinitely without ever having to re-authenticate.

Reply:
Thanks for the explanation! I re-worked this module based on your text - please can you check it carefully.